source/docs/articles/sites/backups/restoring-an-environment-from-a-backup.md

```diff
@@ -23,65 +23,44 @@ When a restore starts, it is placed in a queue and executed. Depending on the si
-
-1. [Clone the site's code](/docs/articles/local/starting-with-git/) or download an archive from the Backups Tool.
-2. From the command line, `cd` to the codebase directory.
-3. Run `git tag` and locate the highest numbered tag associated with the Live environment (e.g. `pantheon_live_114`). The tag with the highest value reflects the current state of the Live environment.
-
- If you wish to restore from a previous state of the Live environment, first confirm the tag points to the desired commit history and restoration point:
-
- ```
- git show [pantheon_live_n]
- ```
-
-4. To preserve the history of your code changes while restoring the environment, run:
-
- ```
- git revert [pantheon_live_n]
- ```
- If you would like to overwrite the environment's codebase and history entirely so that it reflects the state of the code for the given tag, execute:
+### Restore Database and Files
+To restore Database and Files, navigate to the target environment and click the **Workflow** tab. Choose **File** and upload the backups for Database and Files if you downloaded the archives directly, otherwise provide the temporary URL for each backup. Click **Import** for each backup part to restore.
 
- ```
- git reset --hard [pantheon_live_n]
- ```
-
-
-5. Push to Pantheon:
-
- ```
- git push origin master
- ```
+If you have an existing database or file archive that you want to import from an external source, you can also upload the content here.  
+![Workflow Tab](/source/docs/assets/images/desk_images/169632.png)  
 
-If `reset` was used above, you will need to include the `--force` option to push your changes to Pantheon. The `--force` option should be used sparingly, especially in distributed team environments. For more information, see [`git push`](https://git-scm.com/docs/git-push).
+### Restore Code
+Code archives contain the full remote Git repository and reflect the state of code for the given environment. Backups created on the Test and Live environments automatically checkout the [`git tag`](https://git-scm.com/book/en/v2/Git-Basics-Tagging) associated with the most recent deployment. However, if you would like to rewind an environment's codebase to a previous state we recommend using `git revert` or `git reset` instead of a code archive.
 
-## Import Existing Content
+#### Revert Commits and Preserve History
+This method is recommended for distributed teams working collaboratively. To undo commits while preserving the site's Git history:
 
-Once you have the downloaded parts and/or links, navigate to the target environment and click the **Workflow** tab.  
+1. Identify the commit you want to undo using the commit history provided in the site Dashboard or by reviewing `git log` locally
+2. Copy the commit ID
```

Comment:
Consider adding a small screenshot of a commit, highlighting the ID in the dashboard. 
![image](https://cloud.githubusercontent.com/assets/646061/13539027/99ff087e-e202-11e5-8ac2-c32625bf99af.png)